### ISSUE TITLE
jshafer817/vjoy works with Snoothy/UCR but njz3/vjoy doesn't

### ISSUE THREAD

**Issue by freecodecampster** [shauleiz/vJoy#51]
Mentioned here as there is no issues tab on their repository pages

**Comment by jshafer817** [comment]
Have you asked njz3? He did a lot of work on force feedback that I never added because mainly my repository is as close as possible to the original vjoy...  The only reason I bought the code signing cert is because I use VJoy in my own project and so... I need it to work on Windows 10, but all I want is 2 buttons... I don't even care about the axis. =)

**Comment by njz3** [comment]
The vjoy v2.2.x branch (njz3) has a different memory stucture to exchange data with the driver, specifically made to fix a few bugs withing the FFB module. It is not completly backward compatible with the v2.1.x branch.
If you need to use v2.2.x, please update to the new vJoyInterface.dll (new SDK) to make it compatible.

**Comment by cyberluke** [comment]
Yes, it would be nice if njz3 could enable issues tab in Github project settings.

Ah, thanks for mentioning that! I could not wrap my head around why older driver did not work. I'm not fetching VJoy old code even from SVN and need to update njz3 fork, so it will compile Windows XP (retrogaming) again as it worked before.
